product_pack/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,4 @@
+* `Tecnativa <https://www.tecnativa.com>`_:
```

Comment:
You should add here Ing. Adhoc contributors

Reply:
This seems resolved.